src/cloud-api-adaptor/podvm/qcow2/rhel/qemu-rhel.pkr.hcl

```diff
@@ -1,3 +1,12 @@
+packer {
+  required_plugins {
+    qemu = {
+      version = ">= 1.1.0"
```

Comment:
I meant` version = "= 1.1.0"` or `version = "<= 1.1.0"`  because latest may introduce unexpected changes (happened in azure plugin)

Reply:
Done